src/EditorFeatures/Core/Options/LegacyGlobalOptionsWorkspaceService.cs

```diff
@@ -23,22 +23,22 @@ internal sealed class LegacyGlobalOptionsWorkspaceService : ILegacyGlobalOptions
         private readonly CodeActionOptionsStorage.Provider _provider;
 
         private static readonly Option2<bool> s_generateOverridesOption = new(
-            "GenerateOverridesOptions_SelectAll", defaultValue: true);
+            "dotnet_select_all_when_generate_overrides", defaultValue: true);
 
         private static readonly PerLanguageOption2<bool> s_generateOperators = new(
-            "GenerateEqualsAndGetHashCodeFromMembersOptions_GenerateOperators",
+            "dotnet_generate_equals_and_get_hash_code_from_members_when_generate_operators",
             defaultValue: false);
 
         private static readonly PerLanguageOption2<bool> s_implementIEquatable = new(
-            "GenerateEqualsAndGetHashCodeFromMembersOptions_ImplementIEquatable",
+            "dotnet_generate_equals_and_get_hash_code_from_members_when_implement_iequatable",
```

Comment:
>dotnet_generate_equals_and_get_hash_code_from_members_when_implement_iequatable [](http://example.com/codeflow?start=13&length=79)

`dotnet_generate_iequatable_implementation`

Reply:
I will use the first one